<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KnotParamsWidget</class>
 <widget class="QWidget" name="KnotParamsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxParams">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Params</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="yLabel">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="zLabel">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="tensionLabel">
        <property name="text">
         <string>Tension</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xDoubleSpinBox">
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="yDoubleSpinBox">
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="zDoubleSpinBox">
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="tensionDoubleSpinBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="continuityLabel">
        <property name="text">
         <string>Continuity</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="continuityLabel_2">
        <property name="text">
         <string>Bias</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="continuityDoubleSpinBox"/>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="biasDoubleSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
